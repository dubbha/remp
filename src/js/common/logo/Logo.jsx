import React from 'react';
import './style.sass';

const Logo = () => (
  <div className="logo">
    themoviedb
  </div>
);

export default Logo;
