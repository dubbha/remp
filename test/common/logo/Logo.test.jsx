import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Logo from 'common/logo';

describe('Logo', () => {
  it('should render successfully', () => {
    const props = { onSearchClick: jest.fn() };
    const wrapper = shallow(<Logo {...props} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
