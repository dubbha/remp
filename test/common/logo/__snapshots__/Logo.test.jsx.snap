// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render successfully 1`] = `
<div
  className="logo"
>
  themoviedb
</div>
`;
